cerberus/kubernetes/client.py

```diff
@@ -39,18 +39,24 @@ def list_pods(namespace):
 def monitor_nodes():
     nodes = list_nodes()
     notready_nodes = []
+    node_kerneldeadlock_status = None
     for node in nodes:
         try:
             node_info = cli.read_node_status(node, pretty=True)
         except ApiException as e:
             logging.error("Exception when calling \
                            CoreV1Api->read_node_status: %s\n" % e)
         for condition in node_info.status.conditions:
-            if condition.type == "Ready":
-                node_status = condition.status
+            if condition.type == "KernelDeadlock":
```

Comment:
Node problem detector can monitor multiple conditions, let's mention the conditions supported by cerberus in the readme so that user knows that the rest of them are not taken into account when looking for unhealthy nodes.